gsy_e_sdk/setups/grid_operator_api_scripts/redis_aziiz_strategy.py

```diff
@@ -73,7 +73,7 @@ def _set_new_market_fee(self):
                 if MOVING_AVERAGE_PEAK:
                     fees = []
                     # pylint: disable=unused-variable
-                    for j,k in enumerate(self.balance_hist):
+                    for j, k in enumerate(self.balance_hist):
```

Comment:
Why using enumerate if you do not use the number? Can't you just loop over self.balance_hist ? You can remove the ` # pylint: disable=unused-variable` then too